core/chains/evm/evm_txm.go

```diff
@@ -0,0 +1,49 @@
+package evm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/sqlx"
+
+	evmclient "github.com/smartcontractkit/chainlink/core/chains/evm/client"
+	evmconfig "github.com/smartcontractkit/chainlink/core/chains/evm/config"
+	httypes "github.com/smartcontractkit/chainlink/core/chains/evm/headtracker/types"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/logpoller"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/txmgr"
+	evmtypes "github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/logger"
+)
+
+var _ httypes.HeadTrackable = &evmTxm{}
+
+// evmTxm is an evm wrapper over the generic TxManager interface
+type evmTxm struct {
+	httypes.HeadTrackable
```

Comment:
is this redundant? TxManager already requires `txmgrtypes.HeadTrackable[*evmtypes.Head]`

Reply:
Yes, the whole `evmTxm` type is actually not necesssary: https://github.com/smartcontractkit/chainlink/pull/8744

Reply:
Thanks a lot Jordan!